dp/Makefile

```diff
@@ -64,12 +64,17 @@ clean:
 dev_orc8r:
 	skaffold dev -p orc8r-deployment --trigger=manual
 
+
 .PHONY: dev_orc8r_integration_test
-dev_orc8r_integration_test:
+dev_orc8r_integration_test: cleanup_test_db
```

Comment:
Is this "private target", but prefixing it with `_`, i.e. can this be run as standalone command? There was also `orc8r_integration_tests` target which also did some setup required by this one